layouts/stories/single.html

```diff
@@ -57,7 +57,15 @@ <h1 class="f1 mt3 mb1">
   </section>
   <div class="nested-copy-line-height lh-copy  {{ $.Param "post_content_classes"  | default "avenir"}} f4 nested-links nested-img dark-gray pr4-l">
     {{- .Content -}}
-    {{- partial "tags.html" . -}}
+    <div class="tag-group" style="display:inline-block;">
```

Comment:
This is definitely a valid approach, but one of the tradeoffs that you make here is that by moving the modified content of tags.html here, you lose the improvements you've made in any other pages. So should we start a blog on the site, we wouldn't get the benefit of your work.

So to make these modifications to tags.html, what you would do is:

1. Create a `tags.html` file in `layouts/partials/`
1. Move what you've added here to that new `tags.html` file
1. Revert this file

Test it out! Even though the theme still has a `tags.html` file, you will see that the one you've added will take precedent.